_episodes/06-script.md

```diff
@@ -391,9 +391,19 @@ $ cd ../north-pacific-gyre/2012-07-03/
 ```
 {: .language-bash}
 
+> ## Shebangs
+> Take a look at the first line of Nelle's script and see how it starts with the '#!' pattern?
+> This character sequence is called a shebang and it's used to declare the interpretter for the script to use.
```

Comment:
interpretter -> interpreter

Reply:
What's an interpreter? I don't think this is covered anywhere in the lesson.

Reply:
Good point! We should use a term already in the lesson. Can we just use "program"?